ui/StatusQ/src/StatusQ/Components/private/statusMessage/StatusMessageImageAlbum.qml

```diff
@@ -0,0 +1,57 @@
+import QtQuick 2.15
+import QtQuick.Layouts 1.14
+import QtQuick.Controls 2.14
+
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+
+
+RowLayout {
+    id: root
+
+    property var album: []
+    property int albumCount: 0
```

Comment:
Just thinking out loud... why do you need this separate `albumCount` property? Isn't it always `album.length`?

Reply:
cos now album of images is several messages where part can be not delivered but every one contains a count of images in album. As result we can have, for example, `album.lenght == 3` and `albumCount == 5` and display 2 loading components.